usecases/modules/modules.go

```diff
@@ -549,29 +550,24 @@ func (p *Provider) ListObjectsAdditionalExtend(ctx context.Context,
 // GetExploreAdditionalExtend extends graphql api get queries with additional properties
 func (p *Provider) GetExploreAdditionalExtend(ctx context.Context, in []search.Result,
 	moduleParams map[string]interface{}, searchVector []float32,
-	argumentModuleParams map[string]interface{},
```

Comment:
I think that `class *models.Class` parameter is not needed bc `class` definition is taken from `in []search.Result`. Can we remove this param?